### ISSUE TITLE
Upgrade to version v6.0.1 fails

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-backend#670]
During upgrade the following is printed:
```
t/test01.t ................ 11/? Undefined language string: 'fr_FR'
```
The problem will not be seen during a fresh installation.

With Zonemaster-Backend v6.0.1 (v2020.1) the language string definition has changed. From that version it must be set in the backend_config.ini, or else it is not accepted. There is a unit test that depends in "fr_FR" to be defined.

The problem is that the old `/etc/zonemaster/backend_config.ini` does not have that setting, and that will unfortunately be used by the unit test.

Work around:
* Remove `/etc/zonemaster/backend_config.ini` and then copy a new configuration file following the installation instructions. Save a copy if that was not a standard configuration file.

**Comment by matsduf** [comment]
This issue could create problem for future upgrades too if the installed ini file (`/etc/zonemaster/backend_config.ini`) has been changed when it comes to supported languages.

**Comment by matsduf** [comment]
Resolved by #671
